### ISSUE TITLE
Keeping height of thumbnail even

### ISSUE THREAD

**Issue by andiruleu** [sachinchoolur/lightslider#183]
How do I keep the thumbnails even in height? 

![screen shot 2015-11-17 at 9 37 49 pm](https://cloud.githubusercontent.com/assets/1781551/11233537/7f624802-8d73-11e5-94d4-915f3cce68e4.png)

Thanks!

**Comment by SimonEast** [comment]
Yes, we have a similar problem. We've found that we can override the CSS, but the JS code that manages the scrolling then works a bit oddly.

I think the workaround for us is to simply crop the images so that they're all the same heights. Unfortunate workaround though. :-(
